pythonx/netranger/netranger.py

```diff
@@ -2078,18 +2078,24 @@ def _NETRSearchStop(self, accept):
         # clear command line
         Vim.command('echo')
 
-    def _NETRSearchGTNext(self):
-        accept_line_nr = [n.name for n in self._cur_search_buf.nodes
-                          ].index(Vim.current.line) + 1
+    def _NETRSearchGTNext(self,forward):
+        if forward:
+            accept_line_nr = [n.name for n in self._cur_search_buf.nodes
+                              ].index(Vim.current.line) + 1
         
         self._NETRSearchStop(True)
-        nod=self.cur_buf.nodes[accept_line_nr-1]
-        print('zz',nod.name)
-        self.cur_buf.set_clineno_by_node(nod)
-        if nod.is_DIR:
-            self.NETRBufOpen()
-            # Vim.command("norm kztjj")
+        if forward:
+            nod=self.cur_buf.nodes[accept_line_nr-1]
+            self.cur_buf._redraw()
+            self.cur_buf.set_clineno_by_node(nod)
+            # self.cur_buf.update_nodes_and_redraw(force_redraw=True)
+            if nod.is_DIR:
+                self.NETRBufOpen()
+                self.NETRSearch()
+        else:
+            self.NETRParentDir()
```

Comment:
I feel like this is a very niche need. Why would you want to go to parent directory when you search? Search and goto the search target sounds like a plausible need, but search and goto parent sounds like two unrelated things.